src/phoenix/trace/span_json_decoder.py

```diff
@@ -39,14 +29,14 @@ def json_to_attributes(obj: Optional[Dict[str, Any]]) -> Dict[str, Any]:
         obj[INPUT_MIME_TYPE] = MimeType(mime_type)
     if mime_type := obj.get(OUTPUT_MIME_TYPE):
         obj[OUTPUT_MIME_TYPE] = MimeType(mime_type)
-    if documents := obj.get(RETRIEVAL_DOCUMENTS):
-        obj[RETRIEVAL_DOCUMENTS] = [json_to_document(document) for document in documents]
     return obj
 
 
 def json_to_span(data: Dict[str, Any]) -> Any:
     """
     A hook for json.loads to convert a dict to a Span object.
+
+    NB: this function is mainly used for testing purposes. Consider swapping this out for pydantic.
```

Comment:
did you re-add this? I removed it intentionally.

Reply:
it's just from the revert. i can remove